src/utils/testUtils.js

```diff
@@ -32,3 +33,13 @@ export const hasRoleAttribute = (el, roleName) => {
 export const componentHasProperty = (component, prop, value) => {
 	expect(component && component.props && component.props[prop] === value).toBe(true);
 };
+
+// Note: functional components need to be
+// wrapped it in a stateful component to use TestUtils effectively
+export const TestWrapper = createReactClass({
+	render: function() {
```

Comment:
in general the preferred solution would be to use enzyme instead of `TestUtils.renderIntoDocument` in these cases - I think it's fine to add this helper for 'legacy' code, but I'd like to see a `@deprecated` code comment and (ideally) a `console.warn('Using deprecated TestWrapper for TestUtils.renderIntoDocument - covert test to use enzyme')`